### ISSUE TITLE
Interface shows no results

### ISSUE THREAD

**Issue by Johannes1509** [inspectIT/inspectIT#398]
want to use "InspectIT" (a free profiling tool) for my programs. I've tested InspectIT on "sqldeveloper" from Oracle - just to see if its works as it should. My problem: Everytime I recorded a Session the Data Explorer shows no results, only the System Overview is displayed. But I need to know how fast are the methods... Can anybody help me?

Thank you Johannes

Some Images/Screenshots:

https://ibb.co/g2zm1K

https://ibb.co/fOdeMK

https://ibb.co/j8wATz

**Comment by ivansenic** [comment]
Hi, @Johannes1509 

I don't have a clue what "sqldeveloper" is. 

The inspectIT comes with the default instrumentation configuration, which is orientated toward the HTTP applications. Per default, invocations and traces are started on the HTTP requests. Not sure if you have any of those in the "sqldeveloper". You can check the `Instrumentation Browser` tree to understand what classes are instrumented. Not seeing anything only means that the classes/methods instrumented are not invoked. So no results can be perfectly fine. You can add your own configuration profile for the methods you are interested in measuring the exectuion time. 

inspectIT is not a profiler and uses instrumentation to add measurements points. Profilers use sampling, so you usually see a bunch of HOT method.

**Comment by Johannes1509** [comment]
Thank you @ivansenic for spending time on solving this Problem!

I'm sorry, but your solution doesn't work for me.

I have set all checkmarks and profiler to "true"(in the enviroment settings).
But the "sql statments" and all the other monitoring tools (except the "System Overview")show no results. The table is always empty.
++++++++++++++++++++++++++++
Something about "sqldeveloper":
It's just a tool based on java, which I use as an example to test the InspectIT system. The agent-connection is via plain-Java in the config-file of "sqldeveloper".

Have you got an idea? How can I solve the problem?

Best regards 
Johannes Müller

**Comment by ivansenic** [comment]
Are you sure that your tool is executing SQL statements against some database?

**Comment by Johannes1509** [comment]
Yes, and furthermore no results shown in "Invocation Sequences" or "Timer Data". (As I mentioned before) the tool shows only the "System Overview" stats.

**Comment by ivansenic** [comment]
Can you create your own `Profile` as described [here](https://inspectit-performance.atlassian.net/wiki/spaces/DOC18/pages/93009248/Sensor+assignments+configuration), add an instrumentation point for the method that you would like to measure execution time using `Timer sensor`. This should appear then in the `Timer data` view. Let's just check that the instrumentation works. In addition, can you expand the `Instrumentation Browser` and send the picture here so that we can understand what's instrumented?
